gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/AbstractSession.java

```diff
@@ -129,7 +128,7 @@ protected enum CloseReason {
 
         /**
          * The session was interrupted by the channel closing, which can be something initiated by closing the
-         * {@link Client} or might be triggered by the server. This may not be considered an error situation and
+         * Client or might be triggered by the server. This may not be considered an error situation and
```

Comment:
Why was this link removed?

Reply:
When I removed the dependency from gremlin-server on gremlin-driver I got a warning that Client cannot be resolved from server anymore. I looked at it more, it is just an intelij warning. Would I have to build the docs to see if the link is still functional?